src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIMeleeGuard.java

```diff
@@ -163,26 +163,26 @@ protected AIState huntDown()
 
             if (killedEnemy)
             {
-                return AIState.GUARD_GATHERING;
+                return GUARD_GATHERING;
             }
 
             if (attacksExecuted >= getMaxAttacksUntilRestock())
             {
-                return AIState.GUARD_RESTOCK;
+                return GUARD_RESTOCK;
             }
 
-            return AIState.GUARD_HUNT_DOWN_TARGET;
+            return GUARD_HUNT_DOWN_TARGET;
         }
 
         if (shouldReturnToTarget(targetEntity.getPosition(), FOLLOW_RANGE))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "targetEntity" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)